.github/workflows/release.yaml

```
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  push:
+    tags:        
+      - '*' 
+
+jobs:
+
+  rancher-ci:
+    runs-on: ubuntu-latest
+    container:
+      image: rancher/dapper:v0.5.3
+      env:
+        GIT_IN_DAPPER: true
+        DOCKER_USER: ${{ secrets.docker_username }}
+        DOCKER_PASS: ${{ secrets.docker_password }}
+    steps:
+      - name: Add git
+        run: |
+          apk git add
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: tag
+        run: |
+          dapper ci
```

nit: add a blank line at the end of the file
```suggestion
          dapper ci

```